brainscore_language/plugins/pereira2018/ceiling_packaging.py

```diff
@@ -0,0 +1,305 @@
+import logging
+import sys
+import warnings
+from collections import defaultdict
+
+import numpy as np
+from numpy import AxisError
+from numpy.random import RandomState
+from result_caching import store
+from scipy.optimize import curve_fit
+from tqdm import tqdm, trange
+
+from brainio.assemblies import DataAssembly, array_is_element, walk_coords, merge_data_arrays
+from brainscore_core.metrics import Score
+from brainscore_language import load_benchmark
+from brainscore_language.utils import fullname
+from brainscore_language.utils.s3 import upload_data_assembly
+from brainscore_language.utils.transformations import apply_aggregate
+
+_logger = logging.getLogger(__name__)
+
+"""
+Because ceiling estimations for Pereira2018-linear benchmarks are very costly to run (many hours), we pre-compute them,
+and store the resulting ceiling (including per-neuroid estimates) on S3.
+The code in this file was run only once, and is kept here for reference.
+"""
+
+
+def upload_ceiling(experiment):
+    benchmark = load_benchmark(f'Pereira2018.{experiment}-linear')
+    ceiler = ExtrapolationCeiling()
+    ceiling = ceiler(benchmark.data, metric=benchmark.metric)
+    _logger.info(f"Uploading ceiling {ceiling}")
+    # Note that because we cannot serialize complex objects to netCDF, attributes like 'raw' and 'bootstrapped_params'
+    # will get lost during upload
+    upload_data_assembly(ceiling,
+                         assembly_identifier=benchmark.identifier,
+                         assembly_prefix='ceiling_')
+    # also upload raw (and raw.raw) attributes
+    upload_data_assembly(ceiling.raw,
+                         assembly_identifier=benchmark.identifier,
+                         assembly_prefix='ceiling_raw_')
```

Comment:
we need to come up with different names instead of the nested raws, we can talk about this next week

Reply:
could call it `per_neuroid` here? I like `raw` whenever the overall score is computed from it, other attributes have more descriptive names

Reply:
created issue #42